catalog/task/run-file-updates/0.2/README.md

```diff
@@ -0,0 +1,19 @@
+# run-file-udpates
+
+Tekton task to create InternalRequests for each repository that needs to be updated. This information is extracted from
+the field `spec.extraData.fileUpdates` in the ReleasePlanAdmission resource.
+
+## Parameters
+
+| Name            | Description                                                   | Optional | Default value                 |
+|-----------------|---------------------------------------------------------------|----------|-------------------------------|
+| jsonKey         | JSON key where the information is defined                     | Yes      | .spec.extraData.fileUpdates[] |
+| fileUpdatesPath | Path to the JSON file containing the key                      | No       |                               |
+| request         | Type of request to be created                                 | Yes      | file-updates                  |
+| synchronously   | Whether the task should wait for InternalRequests to complete | Yes      | true                          |
+
+## Changelog
+
+### changes since 0.1
+- calls `update-paths` script to run the yq updates for mapped snapshots
```

Comment:
it took me a sec to understand this. Can you say like `run the yq updates to do X on the snapshot` instead?

Reply:
can you take a look if now it reads better?